Atomic/atomic.py

```diff
@@ -1113,6 +1114,29 @@ def set_debug(self):
         if self.args.debug:
             self.debug = True
 
+    def get_last_time_all_scanned(self):
+        """
+        Will read through the scan_summary.json file and get the most recent scan's time.
+        """
+        try:
+            with open(os.path.join(self.results, "scan_summary.json"), "r") as scan_summary:
+                summary_results = json.loads(scan_summary.read())
+            mostRecentTime = ""
+            currentTime = ""
+            for uuid in summary_results.keys():
+                if len(summary_results[uuid].keys()) > 0:
+                    if summary_results[uuid]["All"]:
+                        if not mostRecentTime:
```

Comment:
So upon speaking with Andreas from the Cockpit team, he says that there should be a box displayed on the UI that lists information from when all containers and images were scanned and then display the results from that. Thus, I will be going through the scan_summary.json file and specifically looking for the cases when everything was scanned and take information about the time of the last scan.